index.js

```diff
@@ -247,10 +265,13 @@ class Client {
 
   _uploadFile(filePath) {
     let _this      = this,
-        fileKey    = filePath.replace(_this.clientPath, '').substr(1).replace(/\\/g, '/');
+        fileKey    = filePath.replace(_this.clientPath, '').substr(1).replace(/\\/g, '/'),
+        urlRoot    = regionToUrlRootMap(_this.region);
 
     this.serverless.cli.log(`Uploading file ${fileKey} to bucket ${_this.bucketName}...`);
-    this.serverless.cli.log(`If successful this should be deployed at: https://s3.amazonaws.com/${_this.bucketName}/${fileKey}`)
+    this.serverless.cli.log('If successful this should be deployed at:')
+    this.serverless.cli.log(`https://${urlRoot}/${_this.bucketName}/${fileKey}`)
```

Comment:
This url isn't correct. I'll remove it when I push this version. 

Compare:
http://serverless-finch-test.s3-website-us-east-1.amazonaws.com/ (what is output by line 274)

vs. what is output by this line:
https://s3-website-us-east-1.amazonaws.com/serverless-finch-test/index.html

The second link fails when deploying in us-east-1

A URL that does actually work is 
https://s3.amazonaws.com/serverless-finch-test/index.html

Reply:
Maybe it makes sense to add back your original url format while you're at it.
```
this.serverless.cli.log(`https://s3.amazonaws.com/${_this.bucketName}/${fileKey}`)
```

Reply:
@amsross I could but that's also incorrect when you publish to another region. I prefer line 274. We'll stick with the correct format not the hacky "but it works" one I had before XD

Here's an example of where the line you just posted in that comment would fail. Let's say I deploy a static site to S3 in us-west-2. The working URL (and correct as far as AWS goes) is:
`https://s3-us-west-2.amazonaws.com/serverless-finch-test-nonuseast1test/index.html`

If I use the `https://s3.amazonaws.com/` root all the time then the URL would be
`https://s3.amazonaws.com/serverless-finch-test-nonuseast1test/index.html`
Spoiler - this fails. 

So TLDR there is a region issue in the previous version that was using s3.amazonaws.com

The newest version will fix it by changing to the correct output you put together on line 274. But I will trim out line 273 because it isn't correct.

Reply:
LGTM!